app/helpers/js_helper.rb

```diff
@@ -9,16 +9,6 @@ def set_spinner_off
     'miqSparkleOff();'
   end
 
-  # replacement for app/views/shared/ajax/_tree_lock_unlock.js.erb
-  def tree_lock(tree_var, lock = true)
-    bool_str = (!!lock).to_s
-    element = "#{tree_var}_div"
-    "
-      miqTreeObject('#{j_str(tree_var)}').#{lock ? 'disable' : 'enable'}All({silent: true, keepState: true});
```

Comment:
Burn it with :fire:  :)

Reply:
![](https://static1.fjcdn.com/comments/4148982+_8ef88cd64e6ca94b1b849ecb8dd59063.jpg)